src/libs/saveLastScreen/index.ts

```diff
@@ -0,0 +1,3 @@
+/** we are only saving last screen in case of ios */
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+export default function saveLastScreen() {}
```

Comment:
```suggestion
const saveLastRoute: GetPlaidDesktopMessage = () => {};

export default saveLastRoute;

```

Reply:
@DylanDylann There is a issue of type mismatch for return (Type 'void' is not assignable to type 'TranslationPaths | undefined)

Reply:
@ravindra-encoresky  ahh my bad, please remove GetPlaidDesktopMessage